SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+If you believe you've found something in Django REST Framework JSON API which has security implications, please **do not raise the issue in a public forum**.
+
+Send a description of the issue via email to [whatemailaddressshouldweuse@example.net][security-mail].The project maintainers will then work with you to resolve any issues where required, prior to any public disclosure.
```

Comment:
@sliverc It still shows `whatemailaddressshouldweuse@example.net` here rather than the proper name.

![image](https://user-images.githubusercontent.com/4356471/111332870-89a52180-8648-11eb-8be8-a950a5e3f01f.png)

Reply:
Oops missed that one. Updated now and also fixed some other typos.